Makefile

```diff
@@ -0,0 +1,20 @@
+MODULE = gammapy
+
+CURDIR = $(shell pwd)
+
+build:
+	python $(CURDIR)/setup.py develop
+
+test: build
+	python $(CURDIR)/setup.py test
+
+doc:
+	python $(CURDIR)/setup.py build_sphinx
+
+all: build test doc
```

Comment:
OK. I removed the methods anyway, as mentioned above.
For the records, `build` here means run `python setup.py develop`, which I guess it's not automatically run, when calling `test` or `doc`?